### PR TITLE
Create node-software.md

### PR DESCRIPTION
**Pull request by Phyloiota**
created content

# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancment

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: docs/participate/support-the-network/node-software.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+title: Node Software
+description: IOTA networks consist of interconnected nodes that run the same node software. We introduce the different available Types and provide guides that will help in setting those nodes up.
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:3]:
```suggestion
description: IOTA networks consist of interconnected nodes that run the same node software. We introduce the different available types and provide guides that will help in setting those nodes up.
```

File: docs/participate/support-the-network/node-software.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+The source code of the software can be found here:
+
+- [Hornet GitHub](https://github.com/iotaledger/hornet)
+
+### Hornet Installation guides:
+
+Standard installation following the Hornet Documentation:
+
+- [Hornet documentation - setup a node](https://hornet.docs.iota.org/getting_started/installation_steps.html/)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:34]:
```suggestion
- [Hornet documentation - set up a node](https://hornet.docs.iota.org/getting_started/installation_steps.html/)
```

File: docs/participate/support-the-network/node-software.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+### Hornet Community Tutorials:
+
+Guides and easy one - line installations scripts created by IOTA community members:
+
+* Nuriels playbook installation using Docker: [Nuriel's playbook](https://github.com/nuriel77/hornet-playbook)
+* 
+* DeMicheles one-liner using Nuriels Playbook: [Install Hornet](https://github.com/demichele/install-hornet-1.5)
+* 
+* Step by Step guide for Windows users by Phylo:
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:44]:
```suggestion
* Step-by-Step guide for Windows users by Phylo: 
```

File: docs/participate/support-the-network/node-software.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+Bee will also be further developed to be a reference implementation in the upcoming decentralized IOTA 2.0 Mainnet.
+
+The full documentation for BEE can be found here:
+
+- [BEE Documentation](https://bee.docs.iota.org/)
+
+The source code of the software can be found here:
+
+- [Bee Github](https://github.com/iotaledger/bee)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:71]:
```suggestion
- [Bee GitHub](https://github.com/iotaledger/bee)
```

File: docs/participate/support-the-network/node-software.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+### BEE Instalation guides
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:73]:
```suggestion
### BEE Installation guides
```

File: docs/participate/support-the-network/node-software.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+Standard installation following the Bee Documentation:
+
+- [Setup a BEE Node](https://bee.docs.iota.org/setup_a_node)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:77]:
```suggestion
- [Set up a BEE Node](https://bee.docs.iota.org/setup_a_node)
```

File: docs/participate/support-the-network/node-software.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+
+## Goshimmer
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:80]:
```suggestion
## GoShimmer
```

File: docs/participate/support-the-network/node-software.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+GoShimmer is a prototype node software exploring the implementation of IOTA 2.0. It is a research and engineering project from the IOTA Foundation seeking to evaluate Coordicide concepts by implementing them in a node software.
+
+Due to the prototypical nature of the project, things written today may reflect how things are tomorrow. 
+
+The software is written in Go and is currently the software used in the IOTA 2.0 DevNet where the upcoming IOTA 2.0 version without a coordinator node is tested and optimized in collaboration with a huge amount of community contributors. Goshimmer allows to test and analyze all upcoming features of IOTA and is therefore an essential part of the ongoing IOTA Research.
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:86]:
```suggestion
The software is written in Go and is currently the software used in the IOTA 2.0 DevNet where the upcoming IOTA 2.0 version without a coordinator node is tested and optimized in collaboration with a huge amount of community contributors. GoShimmer allows testing and analyzing all upcoming features of IOTA and is therefore an essential part of the ongoing IOTA Research.
```

File: docs/participate/support-the-network/node-software.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+Goshimmer will not be developed as an upcoming Mainnet node Software.
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:88]:
```suggestion
GoShimmer will not be developed as an upcoming Mainnet node Software.
```

File: docs/participate/support-the-network/node-software.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+
+The full documentation for Goshimmer can be found here:
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:90]:
```suggestion
The full documentation for GoShimmer can be found here:
```

File: docs/participate/support-the-network/node-software.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+
+- [Goshimmer Documentation](https://goshimmer.docs.iota.org/)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:92]:
```suggestion
- [GoShimmer Documentation](https://goshimmer.docs.iota.org/)
```

File: docs/participate/support-the-network/node-software.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+
+The source code of the software can be found here:
+
+- [Goshimmer GitHub](https://github.com/iotaledger/Goshimmer)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:96]:
```suggestion
- [GoShimmer GitHub](https://github.com/iotaledger/goshimmer)
```

File: docs/participate/support-the-network/node-software.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+
+### Goshimmer Installation guides
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:98]:
```suggestion
### GoShimmer Installation guides
```

File: docs/participate/support-the-network/node-software.md [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+
+Standard installation following the Bee Documentation:
+
+- [Goshimmer Node Setup](https://goshimmer.docs.iota.org/docs/tutorials/setup)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:102]:
```suggestion
- [GoShimmer Node Setup](https://goshimmer.docs.iota.org/docs/tutorials/setup)
```

File: docs/participate/support-the-network/node-software.md [hunk 15]
```diff
@@ -0,0 +1,140 @@
...
+
+### Goshimmer community tutorials:
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:104]:
```suggestion
### GoShimmer community tutorials:
```

File: docs/participate/support-the-network/node-software.md [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+
+- Nuriels Goshimmer Playbook: [Nuriel77 Goshimmer](https://github.com/nuriel77/goshimmer-playbook)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:106]:
```suggestion
- Nuriels GoShimmer Playbook: [Nuriel77 GoShimmer](https://github.com/nuriel77/goshimmer-playbook)
```

File: docs/participate/support-the-network/node-software.md [hunk 17]
```diff
@@ -0,0 +1,140 @@
...
+- DeMichelle Shell Script: [Demichelle Goshimmer](https://github.com/demichele/install-goshimmer)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:107]:
```suggestion
- DeMichelle Shell Script: [DeMichelle GoShimmer](https://github.com/demichele/install-goshimmer)
```

File: docs/participate/support-the-network/node-software.md [hunk 18]
```diff
@@ -0,0 +1,140 @@
...
+* Explore stored messages using an HTTP API
+* Store the data you want by modifying incoming messages
+* Filter data to store it how and where you want (work in progress)
+* 
+
+The full documentation for Chronicle Permanodes can be found here:
+
+- [Chronicle Documentation/GitHub](https://github.com/iotaledger/chronicle.rs)
+
+## WASP Smartcontract Node
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:124]:
```suggestion
## WASP Smart Contract Node
```

File: docs/participate/support-the-network/node-software.md [hunk 19]
```diff
@@ -0,0 +1,140 @@
...
+
+Wasp is a node software developed by the IOTA Foundation to run the IOTA Smart Contract Protocol (ISCP in short) on top of the IOTA Tangle.
+
+Wasp is still a work in progress and is purely for development and testing purposes. It runs in combination with Goshimmer Nodes on the IOTA 2.0 Devnet.
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:128]:
```suggestion
Wasp is still a work in progress and is purely for development and testing purposes. It runs in combination with GoShimmer Nodes on the IOTA 2.0 DevNet.
```

File: docs/participate/support-the-network/node-software.md [hunk 20]
```diff
@@ -0,0 +1,140 @@
...
+
+The full documentation for WASP Smartcontract Nodes can be found here:
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/node-software.md:130]:
```suggestion
The full documentation for WASP Smart Contract Nodes can be found here:
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Comment by Phyloiota** [comment]
Lol, you really got me on this one @Dr-Electron    :)

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**